config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -1614,6 +1621,11 @@ def generate_presubmits_e2e():
             always_run=False,
             use_boskos=True,
         ),
+        presubmit_test(
+            name='presubmit-kops-aws-boskos-kubetest2',
```

Comment:
Creating a new presubmit job just to test this out.  This doesn't work with scenarios (as we discussed in kOps office hours), so this is not a scenario test.